src/blocks/gallery-collage/styles/style.scss

```diff
@@ -1,3 +1,4 @@
+.editor-styles-wrapper .wp-block-coblocks-gallery-collage,
```

Comment:
I am not certain the purpose of this change. Shouldn't the `.wp-block-coblocks-gallery-collage {` selector successfully apply to both editor and front-end blocks?

Reply:
The only purpose is to increase CSS specificity to avoid using `!important`